src/ed25519.rs

```diff
@@ -694,12 +692,6 @@ impl PublicKey {
         Ok(PublicKey(CompressedEdwardsY(bits)))
     }
 
-    /// Convert this public key to its underlying extended twisted Edwards coordinate.
-    #[inline]
-    fn decompress(&self) -> Option<EdwardsPoint> {
-        self.0.decompress()
-    }
```

Comment:
Maybe don't remove this? I'm using it in #31.

Reply:
I think it's only used there in the same way that it was used here, so my preference would be to remove it there too.